src/Slices/ServiceCatalog/UI/ServiceItem.tsx

```diff
@@ -27,13 +29,33 @@ interface Props {
 }
 
 export const ServiceItem: React.FunctionComponent<Props> = ({ service }) => {
-  const { routeManager } = useContext(DependencyContext);
+  const { routeManager, commandResolver } = useContext(DependencyContext);
   const rowRefs = useRef<Record<string, HTMLSpanElement | null>>({});
   const [isOpen, setIsOpen] = useState(false);
   const serviceKey = service.name + "-item";
-
+  const trigger = commandResolver.useGetTrigger<"DeleteService">({
+    kind: "DeleteService",
+    name: service.name,
+  });
+  const [isDeleteModalOpen, setIsDeleteModalOpen] = useState(false);
+  const [errorMessage, setErrorMessage] = useState("");
+  const handleModalToggle = () => {
+    setIsDeleteModalOpen(!isOpen);
+  };
+  const onSubmit = async () => {
+    handleModalToggle();
+    const result = await trigger();
+    if (Maybe.isSome(result)) {
+      setErrorMessage(result.value);
+    }
+  };
```

Comment:
can you keep new lines before return statements? It makes the code a bit more readable :)